mne/io/boxy/boxy.py

```diff
@@ -0,0 +1,404 @@
+# Authors: Kyle Mathewson, Jonathan Kuziek <kuziekj@ualberta.ca>
+#
+# License: BSD (3-clause)
+
+from configparser import ConfigParser, RawConfigParser
+import glob as glob
+import re as re
+import os.path as op
+import pandas as pd
```

Comment:
`pandas` is an optional dependency and should be nested. Ideally you wouldn't need it at all, though

Reply:
yes I can replace with native functions like @rob-luke's nirx example once things are working

Reply:
resolving this will address with comment below